rest_framework/throttling.py

```diff
@@ -5,6 +5,7 @@
 
 from django.core.cache import cache as default_cache
 from django.core.exceptions import ImproperlyConfigured
+from utils.throttling_duration_parser import parse_quantity_and_unit
```

Comment:
If this method is going to be in an other package this import must start with `rest_framework.`, otherwise it could be asysly shadowed by other packages (and _utils_ is way too common as name).

```suggestion
from rest_framework.utils.throttling_duration_parser import parse_quantity_and_unit
```